lib/kubernetes-deploy/statsd.rb

```diff
@@ -4,23 +4,41 @@
 
 module KubernetesDeploy
   class StatsD
+    extend ::StatsD
+
+    PREFIX = "KubernetesDeploy"
+
     def self.duration(start_time)
```

Comment:
Isn't `duration` an instrumentation method in the superclass? I think we need to rename this.

Reply:
It appears that `duration` is indeed a private method on `StatsD::Instrument`. Since it's a private method and statically scoped to the statsd-instrument package, I don't foresee this being a problem.